test/views/DeclarationInformationViewSpec.scala

```diff
@@ -155,12 +168,36 @@ class DeclarationInformationViewSpec extends UnitViewSpec with Injector {
       messages must haveTranslationFor("submissions.timeline")
     }
 
-    "contains page header" in {
+    "contain the navigation banner" when {
+      "the Secure Messaging flag is set to 'exports'" in {
```

Comment:
This feature flag is not the one that allows for 'exports' and 'sfus' values. This one can only be 'enabled' or 'disabled'.
It's `SecureMessagingInbox` that accepts 3 values.

So the test description is incorrect.